ui/components/app/confirm-page-container/confirm-page-container.component.js

```diff
@@ -308,7 +308,24 @@ export default class ConfirmPageContainer extends Component {
           )}
           {isSetApproveForAll && (
             <Dialog type="error" className="confirm-page-container__dialog">
-              {t('confirmPageDialogSetApprovalForAll')}
+              {/*
+                TODO: https://github.com/MetaMask/metamask-extension/issues/15745
+                style={{ fontWeight: 'bold' }} because reset.scss removes font-weight from b. We should fix this.
```

Comment:
I don't have bandwidth to propose an update to restore the font-weight for our <b> elements right now. This is a TODO item

see: https://github.com/MetaMask/metamask-extension/issues/15745